instrumentation/httpclient-5.0/src/main/java/com/nr/agent/instrumentation/httpclient50/InstrumentationUtils.java

```diff
@@ -58,6 +58,7 @@ public static void handleUnknownHost(Exception e) {
     }
 
     public static void processResponse(URI requestURI, HttpResponse response, Segment segment) {
+        if (response == null) return;
```

Comment:
Do you know what triggers the response to be null?
Does it still make sense to create the external in that case?
The HttpParameters could be created calling `noInboudHeaders` instead of `inboundHeaders(Response)`.

Reply:
Good question, I'm not sure what causes that.  What are your thoughts on creating an external anyway?